internal/appconfig/testdata/validation-services-2-duplicate.toml

```diff
@@ -0,0 +1,14 @@
+app = "validation"
+
+[[services]]
+internal_port = 8080
+protocol = "tcp"
+processes = ["bash"]
+
+[[services]]
+internal_port = 8080
+protocol = "tcp"
+processes = ["bash"]
```

Comment:
define an additional process (i.e: "app") and add it only to this service.

Reply:
This would then produce the undefined process error (if added only there), maybe it should be another test case.

Reply:
I will be reading some more, you don't have to respond until I get back from the reading.

Reply:
[Processes run in their own Fly Machines](https://fly.io/docs/apps/processes/#:~:text=Every%20defined%20process). And services seem to be "decorators" of processes?
From this point of view, I don't see a problem with having multiple identical `[[services]]` sections besides those being redundant.